### DISCUSSION TITLE
Kairos + Firecracker: Documented use case or examples?

### DISCUSSION THREAD

**Discussion by bbros-dev** [kairos-io/kairos#1707] [Q&A] [answered]
Is it possible to use kairos with Firecracker micro-VMs? Specifically the A/B boot recovery functionality.
Are there any examples or posts describing use with Firecracker?

**Comment by jimmykarily** [comment, marked as answer]
Nobody in the Kairos team has tried that. If you give it a try, we would welcome documentation PRs ;)
